src/Smalot/PdfParser/RawData/RawDataParser.php

```diff
@@ -455,59 +455,67 @@ protected function decodeXrefStream($pdfData, $startxref, $xref = [])
      *
      * @param string $pdfData  PDF data
      * @param array  $xref
-     * @param string $obj_ref  Object number and generation number separated by underscore character
+     * @param string $objRef   Object number and generation number separated by underscore character
      * @param int    $offset   Object offset
      * @param bool   $decoding If true decode streams
      *
      * @return array containing object data
      *
      * @throws Exception if invalid object reference found
      */
-    protected function getIndirectObject($pdfData, $xref, $obj_ref, $offset = 0, $decoding = true)
+    protected function getIndirectObject($pdfData, $xref, $objRef, $offset = 0, $decoding = true)
     {
-        $obj = explode('_', $obj_ref);
-        if (2 != \count($obj)) {
+        // --- build indirect object header
+        // $objHeader = "[object number] [generation number] obj"
+        $objRefArr = explode('_', $objRef);
+        if (2 !== \count($objRefArr)) {
             throw new Exception('Invalid object reference for $obj.');
         }
-        $objref = $obj[0].' '.$obj[1].' obj';
-        // ignore leading zeros
+        $objHeader = $objRefArr[0].' '.$objRefArr[1].' obj';
+
+        // --- check if we are in position
+        // ignore whitespace characters at offset
```

Comment:
Please add all things you ignore there to the comment. As far as I can tell, you at least ignoring:

* tabs
* new lines
* whitespaces